scripts/bootstrap_win.bat

```diff
@@ -0,0 +1,24 @@
+@echo OFF
+cd /D "%~dp0"
+
+:: cd back up to root
+cd ..
+
+:: Grab dependencies
+git submodule update --init
+
```

Comment:
Are you able to add in the patching for `Atlus-Script-Tools` here, as well?

Reply:
Do we need to add the patch? Since it only edits some paths to be Linux-compatible, it feels like it might only cause problems if we implement this for Windows.

Reply:
Right now the patch just adds the system's own separators into some paths, rather than doing anything Linux-specific. So it should basically do nothing for Windows, which is still a fair reason not to add the patch.

I like the idea of keeping patches the same across OSes wherever possible, but maybe we can test this out later when we've actually added use of the library back in.